language-support/ts/daml-react/hooks.test.ts

```diff
@@ -7,7 +7,9 @@
 import React, { ComponentType, useState } from 'react';
 import { renderHook, RenderHookResult, act } from '@testing-library/react-hooks';
 import DamlLedger, { useParty, useQuery, useFetchByKey } from './index';
+import Ledger from '@daml/ledger';
 import { Template } from '@daml/types';
+import {useLedger} from './hooks';
```

Comment:
This should be imported from `./index` like the rest above.

Reply:
changed.